pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -286,6 +287,94 @@ func (a *ofFlowAction) Group(id GroupIDType) FlowBuilder {
 	return a.builder
 }
 
+//  Learn is an action which adds or modifies a flow in an OpenFlow table.
+func (a *ofFlowAction) Learn(id TableIDType, priority uint16, idleTimeout, hardTimeout uint16, cookieID uint64) LearnAction {
+	la := &ofLearnAction{
+		flowBuilder: a.builder,
+		nxLearn:     ofctrl.NewLearnAction(uint8(id), priority, idleTimeout, hardTimeout, 0, 0, cookieID),
+	}
+	la.nxLearn.DeleteLearnedFlowsAfterDeletion()
```

Comment:
do you need to remove this line now that you added `DeleteLearned`?

Reply:
Removed.